### PR TITLE
Use AfterLoad instead of AfterSet on Structs

### PR DESCRIPTION
**Pull request by lunny**
The newest `xorm` provides new processor `AfterLoad`. For performance,
`AfterSet` will be invoked when a field be filled, but `AfterLoad` only is invoked after all fields be filled. And also `AfterLoad` allows a parameter `session *xorm.Session` that you can load data from the same session with the object to reduce memory usage.

### CODE REVIEW

File: models/login_source.go [hunk 1]
```diff
@@ -183,14 +183,10 @@ func (source *LoginSource) BeforeSet(colName string, val xorm.Cell) {
...
 
-// AfterSet is invoked from XORM after setting the value of a field of this object.
-func (source *LoginSource) AfterSet(colName string, _ xorm.Cell) {
-	switch colName {
-	case "created_unix":
-		source.Created = time.Unix(source.CreatedUnix, 0).Local()
-	case "updated_unix":
-		source.Updated = time.Unix(source.UpdatedUnix, 0).Local()
-	}
+// AfterLoad is invoked from XORM after setting the value of a field of this object.
```
Comment (ethantkoenig) [inline models/login_source.go]: Comment seems stale, please update it (also in other places)

**Review by ethantkoenig — APPROVED** [review]
Looks good, just one suggestion

File: models/login_source.go (hunk 1 above)
Comment (lunny) [reply models/login_source.go]: done.

**Comment by lunny** [comment]
And @ethantkoenig, could you help me update the latest xorm since my `govendor` has some issue.

**Comment by ethantkoenig** [comment]
@lunny I don't have write permissions, so I don't think I can merge into your branch

**Comment by ethantkoenig** [comment]
I could create a separate PR that just updates xorm if that'd be helpful

**Comment by lafriks** [comment]
@lunny fixed govendor hash

**Comment by lafriks** [comment]
LGTM

**Comment by ethantkoenig** [comment]
LGTM

**Pull request merged**